docs/source/backup/index.rst

```diff
@@ -1,272 +1,255 @@
 .. _backup:
 
-On this page you will find useful information on how to backup your database, how to move it to a different location, and how to backup your repository.
+AiiDA stores information in:
 
+ * the ``.aiida`` folder, typically located at ``~/.aiida`` (see also :ref:`configure_aiida`).
+   This contains the configuration of the profiles as well as their associated file repositories.
```

Comment:
Just wanted to point out that technically the file repositories do not have to live within `.aiida`. I think it is probably beter to make the repositories a separate entry in this itemization and then maybe put in parentheses that by default they often are located in the `.aiida` folder.

Reply:
adapted